rollup-http/rollup-http-server/src/rollup/mod.rs

```diff
@@ -258,7 +258,9 @@ pub fn rollup_read_advance_state_request(
         app_contract: Default::default(),
         block_number: 0,
         block_timestamp: 0,
-        prev_randao: Default::default(),
+        prev_randao: { cmt_u256_t {
```

Comment:
```suggestion
        prev_randao: { cmt_abi_u256_t {
```

Reply:
Should we use EVM instead of ABI? Or even EVMABI...

Reply:
This library was going to be called `evm_...` originally, so `evm_abi_...` in this case. But you know how naming things go...

> We can keep ABI and change it later when something else finally appears.
ok